### PR TITLE
adding code that generates all posible quizzes

### PR DESCRIPTION
**Pull request by Frexor**

### CODE REVIEW

File: src/Main.java [hunk 1]
```diff
@@ -1,19 +1,31 @@
...
-        for (String quiz : quizzes) {
-            System.out.println(quiz);
+        Map<String, Set<String>> quizzes = new HashMap<>();
+        equationGenerator.generateCorrectEquations();
+        String[] correctEquations = equationGenerator.correctEquations;
+        for (String correctEquation : correctEquations) {
+            for (String quiz : equationGenerator.allPossibleMatchMoves(correctEquation)) {
+                if (!quizzes.containsKey(quiz)) {
+                    Set<String> newSet = new HashSet<>();
+                    quizzes.put(quiz, newSet);
```
Comment (piekaa) [inline src/Main.java]:
```
Set<String> newSet = new HashSet<>();
quizzes.put(quiz, newSet);
```
can be simplified to 
```
quizzes.put(quiz, new HashSet<>());
```
then there is no need to think about variable name and it's shorter

File: src/Main.java [hunk 2]
```diff
@@ -1,19 +1,31 @@
...
+                }
+                Set<String> s = quizzes.get(quiz);
```
Comment (piekaa) [inline src/Main.java]:
```
Set<String> s = quizzes.get(quiz);
s.add(correctEquation);
```
can be simplified to 
```
quizzes.get(quiz).add(correctEquation);
```

BTW: s is not good variable name

**Review by piekaa — CHANGES_REQUESTED** [review]
Merge previous pull request and resolve conflicts if any

File: src/Main.java (hunk 1 above)
Comment (Frexor) [reply src/Main.java]: Done.

File: src/Main.java (hunk 2 above)
Comment (Frexor) [reply src/Main.java]: Done.

**Review by piekaa — APPROVED** [review]

**Pull request merged**